app/controllers/api/base_controller/parser.rb

```diff
@@ -72,7 +72,7 @@ def validate_api_parameters
       def parse_href(href)
         if href
           path = href.match(/^http/) ? URI.parse(href).path.sub(/\/*$/, '') : href
-          path = "#{@prefix}/#{path}" unless path.match(@prefix)
+          path = "/api/#{path}" unless path.match("/api")
```

Comment:
That's fine for now.  This is the only place (other than the routes.rb of course) where we have /api.  If this ever needs changing it will be easy to do. Thanks.